scipy/stats/_new_distributions.py

```diff
@@ -417,6 +433,10 @@ def _moment_central_formula(self, order, ab, **kwargs):
 
     _moment_central_formula.orders = [2]  # type: ignore[attr-defined]
 
+    def _lmoment_formula(self, order, *, a, b, ab, **kwargs):
+        lmoments = {1: 0.5*(a + b), 2: ab / 6, 3: 0, 4: 0}
```

Comment:
Fun fact: The reason that every uniform L-moment $r>2$ is 0, is because the uniform quantile function is a polynomial of order 1; so it disappears after taking the derivative twice :)